Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/OpenSAML2ComponentBuilder.java

```diff
@@ -321,6 +349,54 @@ private static SubjectConfirmationData createSubjectConfirmationData(final Publi
     }
 
     /**
+     * Creates the subject confirmation data sender-vouches.
+     *
+     * @param certificate the certificate
+     * @param publicKey the public key
+     * @return the subject confirmation data
+     * @throws SAMLComponentBuilderException the exception
+     */
+    private static SubjectConfirmationData createSvSubjectConfirmationData() throws SAMLComponentBuilderException {
+        SubjectConfirmationData subjectConfirmationData = null;
+        try {
+            final SubjectConfirmationDataBean subjectConfirmationDataBean = new SubjectConfirmationDataBean();
+            final KeyInfoBean keyInforBean = null;
```

Comment:
why instantiate keyInforBean ? Can we just pass null in SAML2ComponentBuilder.createSubjectConfirmationData()